packages/shuttle_http/shuttle_http.0.8.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Async library for HTTP/1.1 servers"
+description:
+  "Shuttle_http is a low level library for implementing HTTP/1.1 web services in OCaml."
+maintainer: ["Anurag Soni <anurag@sonianurag.com>"]
+authors: ["Anurag Soni"]
+license: "MIT"
+tags: ["http-server" "http" "http1.1" "async"]
+homepage: "https://github.com/anuragsoni/shuttle"
+doc: "https://anuragsoni.github.io/shuttle/"
+bug-reports: "https://github.com/anuragsoni/shuttle/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "shuttle" {= version}
+  "ppx_jane" {with-test}
+  "core_unix" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/anuragsoni/shuttle.git"
+url {
+  src:
+    "https://github.com/anuragsoni/shuttle/releases/download/0.8.0/shuttle-0.8.0.tbz"
+  checksum: [
+    "sha256=ca742c54ba593759e7d112b76d77f97c9fd279a3c4dbb69d54681e0941d1cff3"
+    "sha512=ecad64f99e9801a56c8e923b736086b693f2e302cb105ed6963a01fdb58503dc37b16d5e6d0c1e3f6851cd2598d4ba9543deb7a4e27904423386ee308fb65431"
+  ]
+}
+x-commit-hash: "873c66738acd4161b3bd73486b5f039db9ca72be"
+available: [ arch = "x86_64" | arch = "arm64" ]
```

Comment:
I will take care of this upstream in a future commit. I'll either reproduce this restriction, or work on fixing builds on 32 bit systems.